lib/main.rb

```diff
@@ -0,0 +1,191 @@
+# lib/main.rb
+# this is called the "driver code"
+
+require "awesome_print"
+require_relative "planet"
+require_relative "solar_system"
+require "csv"
+
+# Page formatting:
+SCREEN_WIDTH = 125
+
+def main
+  # This stuff initializes planet and solar system
+  # Do I want this to be user-driven?
+  mini_star = SolarSystem.new("Mini-Star")
+  import_planets_to_solar_system("../data/planets.csv", mini_star)
```

Comment:
The file path should be relative to the project homepage, so this should be:
`import_planets_to_solar_system("data/planets.csv", mini_star)`